### DISCUSSION TITLE
Adjust background color

### DISCUSSION THREAD

**Discussion by CobreDev** [elrumo/macOS_Big_Sur_icons_replacements#1221] [Help with the website]
I'm messing around with creating a costom icon in Photoshop, but I can't figure out how to change the background color of the template to a custom color, it's just white.

What am I missing?

(Not sure if this is the right place to post it, so I can move it or something if needed)

**Comment by rodgerwerner** [comment]
Double Click the RED layer "Add your icon above."
This will open the linked psb file and show an adjustment layer for BG Fill. Change the colour by double clicking on that adjustment ... then, CMD+S to save the linked psb and load back into the template automatically. At this point you can safely close the psb window, unless you want to make further changes,
